persistence-jdbc/javadsl/src/test/scala/com/lightbend/lagom/javadsl/persistence/jdbc/JdbcReadSideSpec.scala

```diff
@@ -18,17 +19,24 @@ class JdbcReadSideSpec extends JdbcPersistenceSpec with AbstractReadSideSpec {
   lazy val readSide = new JdbcTestEntityReadSide(session)
   lazy val queries = PersistenceQuery(system).readJournalFor[JdbcReadJournal](JdbcReadJournal.Identifier)
 
-  override def eventStream[Event <: AggregateEvent[Event]](aggregateTag: AggregateEventTag[Event], fromOffset: Offset): Source[Pair[Event, Offset], NotUsed] = {
-    val tag = aggregateTag.tag
-    val offset = fromOffset match {
-      case Offset.NONE          => 0l
-      case seq: Offset.Sequence => seq.value() + 1
-      case other                => throw new IllegalArgumentException(s"JDBC does not support ${other.getClass.getSimpleName} offsets")
-    }
-    queries.eventsByTag(tag, offset)
-      .map { env => Pair.create(env.event.asInstanceOf[Event], Offset.sequence(env.offset)) }
-      .asJava
-  }
+  //  override def eventStream[Event <: AggregateEvent[Event]](aggregateTag: AggregateEventTag[Event], fromOffset: Offset): Source[Pair[Event, Offset], NotUsed] = {
+  //    val tag = aggregateTag.tag
+  //    val offset = fromOffset match {
+  //      case Offset.NONE          => 0l
+  //      case seq: Offset.Sequence => seq.value() + 1
+  //      case other                => throw new IllegalArgumentException(s"JDBC does not support ${other.getClass.getSimpleName} offsets")
+  //    }
+  //    queries.eventsByTag(tag, offset)
+  //      .map { env => Pair.create(env.event.asInstanceOf[Event], Offset.sequence(env.offset)) }
+  //      .asJava
+  //  }
+
```

Comment:
I'll clean up the commented code above and the `null` below, and make the equivalent change to the Scala DSL version and the Cassandra read side specs before merging. For now, this is just demonstrating the issue.